package.json

```diff
@@ -1,19 +1,22 @@
 {
-	"name": "todo-app",
-	"version": "1.0.0",
-	"description": "",
-	"main": "index.js",
-	"scripts": {
-		"test": "echo \"Error: no test specified\" && exit 1",
-		"start": "node index.js"
-	},
-	"author": "Yehoshua Tarshish",
-	"license": "ISC",
-	"dependencies": {
-		"bcrypt": "^5.1.0",
-		"dotenv": "^16.3.1",
-		"express": "^4.18.2",
-		"jsonwebtoken": "^9.0.0",
-		"mongoose": "^7.3.0"
-	}
+  "name": "todo-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node index.js",
+    "dev": "nodemon server.js"
+  },
+  "author": "Yehoshua Tarshish",
+  "license": "ISC",
+  "dependencies": {
+    "bcrypt": "^5.1.0",
+    "cookie-parser": "^1.4.6",
+    "dotenv": "^16.3.1",
+    "express": "^4.18.2",
+    "jsonwebtoken": "^9.0.0",
+    "mongoose": "^7.3.0",
+    "nodemon": "^2.0.22"
```

Comment:
`nodemon` should be under dev-dependencies.. 
please uninstall this package
then install it like this:

```npm install nodemon --save-dev```